torch/csrc/jit/tensorexpr/half_support.h

```diff
@@ -0,0 +1,71 @@
+#include <torch/csrc/jit/tensorexpr/ir.h>
+#include <torch/csrc/jit/tensorexpr/ir_printer.h>
+#include <torch/csrc/jit/tensorexpr/ir_visitor.h>
+#include <torch/csrc/jit/tensorexpr/tensor.h>
+
+namespace torch {
+namespace jit {
+namespace tensorexpr {
+
+// Walk the Statment looking for Half size loads/stores.
```

Comment:
Probably best to do in a follow-up diff but it bugs me that something called "Checker" actually does mutation, and the doc comment doesn't give any indication of what it's doing either.  It'd be nice to rename and add a clearer comment.

Reply:
This comment (and your other comments) will be are addressed when I rebase onto  by Nick's PR https://github.com/pytorch/pytorch/pull/47448

Reply:
Yeah this is fixed already :) (you're right though)